charts/seed-operatingsystemconfig/downloader/templates/osc.yaml

```diff
@@ -14,6 +14,9 @@ metadata:
 spec:
   type: {{ required "type is required" .Values.type }}
   purpose: {{ required "purpose is required" .Values.purpose }}
+  {{- if .Values.enableContainerD }}
```

Comment:
Can you add the new Helm chart values to the `values.yaml` so that one can see which values are possible?

Reply:
Ok i got you,
I will add it.